avaframe/in3Utils/geoTrans.py

```diff
@@ -276,115 +276,121 @@ def remeshData(rasterDict, cellSizeNew, remeshOption="griddata", interpMethod="c
 
-    IOf.writeResultToAsc(remeshedDEM["header"], remeshedDEM["rasterData"], outFile, flip=True)
-    log.info("Saved remeshed DEM to %s" % outFile)
-    pathDem = str(pathlib.Path("DEMremeshed", outFile.name))
+    IOf.writeResultToAsc(remeshedRaster["header"], remeshedRaster["rasterData"], outFile, flip=True)
+    log.info("Saved remeshed raster to %s" % outFile)
+    pathRaster = str(pathlib.Path("remeshedRasters", outFile.name))
 
-    return pathDem
+    return pathRaster
 
 
-def searchRemeshedDEM(demName, cfgSim):
-    """search if remeshed DEM with correct name and cell size already available
+def searchRemeshedRaster(rasterName, cfgSim, typeIndicator="DEM"):
+    """search if remeshed raster with correct name and cell size already available
 
     Parameters
     -----------
-    demName: str
-        name of DEM file in Inputs/
+    rasterName: str
+        name of raster file in Inputs/
+    typeIndicator: str
+        which type of raster to handle, possible options Raster or RELTH
     cfgSim: configparser object
         configuration settings: avaDir, meshCellSize, meshCellSizeThreshold
 
     Returns
     --------
-    remshedDEM: dict
-        dictionary of remeshed DEM if not found empty dict
-    DEMFound: bool
+    pathRaster: pathlib path
+        to Raster
+    rasterFound: bool
         flag if dem is found
-    allDEMNames: list
-        of all names of dems found in Inputs/DEMremeshed
+    allRasterNames: list
+        of all names of dems found in Inputs/remeshedRasters
     """
 
-    # path to remeshed DEM folder
-    pathToDems = pathlib.Path(cfgSim["GENERAL"]["avalancheDir"], "Inputs", "DEMremeshed")
-    DEMFound = False
-    pathDem = ""
-    allDEMNames = []
+    # path to remeshed Raster folder
+    pathToRasters = pathlib.Path(cfgSim["GENERAL"]["avalancheDir"], "Inputs", "remeshedRasters")
+    rasterFound = False
+    pathRaster = ""
+    allRasterNames = []
 
     # fetch info on desired meshCellSize
     meshCellSize = float(cfgSim["GENERAL"]["meshCellSize"])
     meshCellSizeThreshold = float(cfgSim["GENERAL"]["meshCellSizeThreshold"])
 
-    # check if DEM is available
-    if pathToDems.is_dir():
-        # look for dems and check if cellSize within tolerance and origin matches
-        demFiles = list(pathToDems.glob("*.asc"))
-        allDEMNames = [d.name for d in demFiles]
-        for demF in demFiles:
-            headerDEM = IOf.readASCheader(demF)
-            if abs(meshCellSize - headerDEM["cellsize"]) < meshCellSizeThreshold and demName in demF.stem:
-                log.info("Remeshed DEM found: %s cellSize: %.5f" % (demF.name, headerDEM["cellsize"]))
-                DEMFound = True
-                pathDem = str(pathlib.Path("DEMremeshed", demF.name))
+    # check if Raster is available
+    if pathToRasters.is_dir():
+        # look for rasters and check if cellSize within tolerance and origin matches
+        rasterFiles = list(pathToRasters.glob("*remeshed%s*.asc" % typeIndicator))
```

Comment:
is this a bit strong requiring it to be called remeshedTypeIndicator and also in line 380 to have rasterName inside the file name?